Dockerfile

```diff
@@ -18,12 +18,44 @@ ARG TARGETOS TARGETARCH
 # Build
 RUN --mount=type=cache,target=/go \
 	CGO_ENABLED=0 GOOS="${TARGETOS}" GOARCH="${TARGETARCH}" make _build
+# Prepare CA certificates
+FROM golang:1.23 AS certs
```

Comment:
Do we need a separate stage for this, or could we just copy it into distroless in the same spot?